internal/proto/daemon.proto

```diff
@@ -53,6 +58,10 @@ message GitRepo {
 //     repeated ServiceStatus statuses = 1;
 // }
 
-message CreateServiceReply {}
+message CreateServiceReply {
+    oneof body {
+        Output output = 1;
```

Comment:
is there a reason for this type switch? what might it be besides an `Output` object?

Reply:
Theoretically before we were returning a `CreateServiceReply` that happened to contain no fields. This was an attempt to force myself to write the code so that we now kinda theoretically have either an `Output` or a non-output reply, but then I removed the non-output reply, since CreateServiceReply doesn't actually return anything, which leaves things weird. I'll undo that.